compiler-core/src/build.rs

```diff
@@ -572,6 +572,18 @@ impl Origin {
     }
 }
 
+impl Display for Origin {
```

Comment:
Don't ever implement traits please! Prefer regular methods are they are much clearer.

Reply:
indeed! added a `.folder_name` method to `Origin`

thanks for the feedback